addons/binding/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/json/Status.java

```diff
@@ -0,0 +1,354 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.volvooncall.internal.json;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.OpenClosedType;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link Status} is responsible for storing
+ * informations returned by vehicule status rest answer
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+public class Status {
```

Comment:
This seem to be a dto (data transfer object).
For the sake of code and review size, could you please remove getters/setters from dtos?
It also has been established as pattern to have dto's in a package "dto" or "model", can you please move those files?
Thanks